src/main/resources/templates/non-strategic/rptEasternWeeklyHearingList.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
```

Comment:
We should find a way for these 5 lists to share a common HTML. They are essentially the same list. If we do it this way, next time we update something need to update 5 times

Reply:
as discussed, it done. Only one template file will be used.